ecosystem/sep-0038.md

```diff
@@ -35,6 +39,49 @@ All endpoints accept in requests the following `Content-Type`:
 All endpoints respond with content type:
 - `application/json`
 
+### Errors
+
+If an error occurs when calling any endpoint, an appropriate HTTP status code
+will be returned along with an error response.
+
+#### Status Code
+
+Common HTTP status codes may be returned for a server. In particular the following are expected:
+
+Status Code | Name | Reason
+-----|------|------------
+`400` | Bad Request | The request is invalid in any way.
+`403` | Permission Denied | No `Authorization` header has been provided or the contents of the header are not accepted as valid.
+
+The response body must contain a human-readable description of the reason for error:
+
+```json
+{
+  "error": "The requested asset is not supported. See GET /prices for supported assets."
+}
+```
+
+### Asset Identification Format
+
+This protocol can be used to provide quotes for any class of asset in exchange for a Stellar asset. The following format is used to identify an asset in API requests and responses.
+
+```
+namespace:symbol:issuer
```

Comment:
❗ I think you should define the format as the scheme and identifier, then for each known scheme define the identifier. It helps make this clearer to separate the definitions and layers. Scheme is the term used by URIs for that initial namespace and it helps to use the same terms.

```suggestion
<scheme>:<identifier>
```